### DISCUSSION TITLE
寶可夢中心

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1778] [General]
台灣首間寶可夢中心在12月8日在台北開幕了！地點在台北信義新光三越A11。作為一名到地的台灣人，剛開幕當然就是排爆，首日就有超過1500人排隊，並且入場整理券甚至在早上就8點就發放完畢，雖然現在人潮隨著時間有漸漸下降了一些，但可見寶可夢在台灣的高人氣。
我上次對於「神奇寶貝」這麼著迷可能得要回朔到小學時代了，從不知道什麼時候他竟然更名成為了「寶可夢」，儘管皮卡丘還是一樣的可愛，但也為我的童年添上了一絲陌生的氣息。相信未來的某一天，當我走在台北寶可夢中心裡的時候，我能夠找回那份童趣和美好回憶。

**Comment by William08290829** [comment]
我自己覺得這寶可夢中心做得算是蠻漂亮的，而且儘管已經開幕快半年，現在假日人潮還是不少，連我這個只有小時候有看過寶可夢，然後長大偶爾玩過寶可夢go的人都會忍不住進去參觀看看。裡面有幾隻娃娃是真的蠻可愛的，雖然我都叫不出名字ww。

**Comment by wanka2030** [comment]
身為熱愛「神奇寶貝」的粉絲，還記得當天就去排寶可夢中心，但最後也只是路過而已，現場的人潮真的太多人了，但現在過去反而人會比較少（但在週末的時候還是有很多人），但裡面宛如電玩中的寶可夢商店，裡面的吊飾、商品應用盡全，還是很值得排出時間去逛一逛👌

**Comment by jeff110303549** [comment]
寶可夢中心在台北開幕的消息確實讓許多粉絲激動不已！這個寶可夢主題的活動中心將為粉絲們帶來許多樂趣和回憶。我自己也是一個蠻大的寶可夢粉絲，即使已經長大了，重溫童年的回憶和追逐寶可夢的足跡也是一件令人期待的事情。希望你能在台北寶可夢中心中度過一個充滿樂趣和美好回憶的時光，有機會我也會去看看的
